client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SecurityDocumentationIT.java

```diff
@@ -1185,41 +1185,49 @@ public void testGetSslCertificates() throws Exception {
             assertThat(certificates.size(), Matchers.equalTo(9));
             final Iterator<CertificateInfo> it = certificates.iterator();
             CertificateInfo c = it.next();
-            assertThat(c.getSerialNumber(), Matchers.equalTo("c0ea4216e8ff0fd8"));
-            assertThat(c.getPath(), Matchers.equalTo("testnode.jks"));
-            assertThat(c.getFormat(), Matchers.equalTo("jks"));
-            c = it.next();
-            assertThat(c.getSerialNumber(), Matchers.equalTo("b8b96c37e332cccb"));
             assertThat(c.getPath(), Matchers.equalTo("testnode.crt"));
             assertThat(c.getFormat(), Matchers.equalTo("PEM"));
+            assertThat(c.getSerialNumber(), Matchers.equalTo("b8b96c37e332cccb"));
```

Comment:
The changes in this test are due to two other changes.

X-Pack had a behaviour (which has been replicated into libs/ssl-config) that would have special case behaviour when you configured a keystore, but did not configure any trust anchors.
In that case it would look for any trusted certs in the keystore, and trust them along with the JDK default truststore.
However, there was a bug that meant it did not include those trusted certs in the certificates API.
ssl-config does not have this bug, so there are now more certs shown in some cases.

There was also an issue that the order of the listed certs was undefined, but consistent based on the underlying keystore. So tests that ran against a fix config would produce a consistent (ordered) output, but not based on any intentional login, just based on random(ish) behaviour.
That's been changed now (in a prior PR) so ssl-config now orders the certs consistently.

This test has been updated to reflect that new behaviour.